master/internal/db/postgres_cluster.go

```diff
@@ -131,3 +134,17 @@ func (db *PgDB) UpdateResourceAllocationAggregation() error {
 	}
 	return nil
 }
+
+// CheckIfRPUnbound checks to make sure both of the specified resource pools are not bound to any
+// workspaces and returns an error if they are.
+func (db *PgDB) CheckIfRPUnbound(defaultAux, defaultCompute string) error {
```

Comment:
don't add to PgDB

Reply:
thought I needed to put it in there for some reason... will move this into the rp workspace bindings db code